README.md

```diff
@@ -1,5 +1,5 @@
 1. Replace `<your_account>` with your Github username in the link
-    - [DEMO LINK](https://<your_account>.github.io/js_2048_game/)
+    - [DEMO LINK](https://Solovei-Andrii.github.io/js_2048_game/)
```

Comment:
Consider adding a link to the PR description

Reply:
not fixed 
![image](https://github.com/mate-academy/js_2048_game/assets/70009205/2c1028da-f0c1-430e-9f93-8715b4f14f35)
![image](https://github.com/mate-academy/js_2048_game/assets/70009205/d86c5f86-7269-4c48-9351-001265160a37)